.travis.yml

```diff
@@ -4,4 +4,7 @@ script:
 language: node_js
 
 node_js:
-  - 0.10
+  - "0.10"
+  - "iojs"
```

Comment:
Hmm, In that way and since we don't have environment features (like streams or dunno) it's pretty viable. Just iojs != node 0.12.7, but iojs 3.x is almost merged to node 4.x (afaik).

:+1: